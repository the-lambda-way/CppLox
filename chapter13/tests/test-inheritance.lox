var NotAClass = "I am totally not a class";

class Subclass < NotAClass {} // ?!
