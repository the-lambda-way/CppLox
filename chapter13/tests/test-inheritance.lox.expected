Superclass must be a class.
[line 3]
